doc/debian/jitsi-meet-turn/turnserver.conf

```diff
@@ -0,0 +1,13 @@
+lt-cred-mech
+use-auth-secret
+keep-address-family
+static-auth-secret=__turnSecret__
+realm=jitsi-meet.example.com
+listening-port=4445
+cert=/etc/jitsi/meet/jitsi-meet.example.com.crt
+pkey=/etc/jitsi/meet/jitsi-meet.example.com.key
+
+no-tcp
```

Comment:
why?

Reply:
Cause we use only turns?

Reply:
Here is our turn config in prosody:
  { type = "stun", host = "jitmeet.example.com", port = "443" },
  { type = "turn", host = "jitmeet.example.com", port = "443", transport = "udp" },
  { type = "turns", host = "jitmeet.example.com", port = "443", transport = "tcp" }

Reply:
Aha. Is there a good readon to use TURNS instead of TURN over TCP?

Reply:
> Aha. Is there a good readon to use TURNS instead of TURN over TCP?

Restrictive firewalls with packet inspection, that allow only ssl connections on port 443, isn't the whole turn server thing because of those?